app/models/service_provider.rb

```diff
@@ -70,11 +70,24 @@ def skip_encryption_allowed
     @allowed_list.include? issuer
   end
 
+  def identity_proofing_allowed?
+    ial.present? && ial >= 2
+  end
+
+  def ialmax_allowed?
+    IdentityConfig.store.allowed_ialmax_providers.include?(issuer)
+  end
+
   def biometric_ial_allowed?
     IdentityConfig.store.biometric_ial_enabled &&
       IdentityConfig.store.allowed_biometric_ial_providers.include?(issuer)
   end
 
+  def semantic_authn_contexts_allowed?
+    IdentityConfig.store.feature_valid_authn_contexts_semantic_enabled &&
```

Comment:
The name feels awkward to me. Would `feature_semantic_acrs_enabled` suffice?

Reply:
That's a good point, but I wanted to make sure that the `normalize_yaml` task sorted this next to the original `valid_authn_contexts` key.